test/clj/rems/email/test_core.clj

```diff
@@ -76,3 +80,25 @@
               "should skip the message quietly"))
         (is (nil? @message-atom))
         (reset! message-atom nil)))))
+
+(deftest test-send-email!-mock-smtp
+  ;; run one case with an smtp server to make sure the Auto-Submitted header is set
+  (let [port 3025
+        server (GreenMail. (ServerSetup. port nil ServerSetup/PROTOCOL_SMTP))]
```

Comment:
Nice testing emails!

Reply:
Yeah this GreenMail was a nice find, I had never heard of it before.